features/custom_matchers/define_matcher.feature

```diff
@@ -335,3 +335,37 @@ Feature: define matcher
       | 4 examples, 2 failures               |
       | expected 9 to be a multiple of 2     |
       | expected 9 not to be a multiple of 3 |
+
+  Scenario: using a method from the EnglishPhrasing module
+    Given a file named "matcher_english_spec.rb" with:
+    """ruby
+      require 'rspec/expectations'
+
+      class Food; end
+      class Fruit < Food; end
+      class Banana < Fruit; end
+      class Carrot < Food; end
+
+      RSpec::Matchers.define :descend_from do |expected|
+        match do |actual|
+          expected.all? { |ancestor| actual.is_a?(ancestor) }
+        end
+        description do
+          "descend from#{RSpec::Matchers::EnglishPhrasing.list(expected)}"
+        end
+      end
+
+      RSpec.describe Banana do
+        it { is_expected.to descend_from([Fruit, Food, Object]) }
+      end
+
+      RSpec.describe Carrot do
+        it { is_expected.not_to descend_from([Fruit, Food, Object]) }
+      end
+      """
+    When I run `rspec ./matcher_english_spec.rb --format documentation`
+    Then the exit status should be 0
+    And the output should contain all of these:
+      | 2 examples, 0 failures                          |
+      | should descend from Fruit, Food, and Object     |
+      | should not descend from Fruit, Food, and Object |
```

Comment:
I'm still in favor of this scenario being removed, given the spec coverage you added.